Validation/RecoTrack/python/plotting/plotting.py

```diff
@@ -1964,7 +1964,7 @@ def _doStats(h, col, dy):
             st.SetX1NDC(startingX)
             st.SetX2NDC(startingX+0.3)
             st.SetY1NDC(startingY+dy)
-            st.SetY2NDC(startingY+dy+0.15)
+            st.SetY2NDC(startingY+dy+0.25)
```

Comment:
This change increases the height of the stat box, right? IIRC the space between stat boxes of different histograms/graphs was already rather small. Can you show that this change does not lead the stat boxes to overlap?

Reply:
From your example I see that your case has indeed more lines in the stat box than MTV has (by default). I'm not against that, but then the height should be made e.g. configurable or automated, and the height should propagate to the "line height" (magical `dy -= 0.19` below) as well.